apps/calendly/src/database/schema.ts

```diff
@@ -0,0 +1,11 @@
+import { uuid, text, timestamp, pgTable } from 'drizzle-orm/pg-core';
+
+export const Organisation = pgTable('organisation', {
+  id: uuid('id').notNull().primaryKey(),
+  accessToken: text('access_token').notNull(),
+  refreshToken: text('refresh_token').notNull(),
+  organizationUri: text('organization_uri').notNull(),
```

Comment:
```suggestion
  organizationId: text('organization_id').notNull(),
```

Is it an id or a uri ?